### ISSUE TITLE
Grid column menu keyboard navigation does not work with custom menu template

### ISSUE THREAD

**Issue by V-Huza** [telerik/kendo-angular#4731]
### Description

I'm trying to implement a custom template for grid column menu. I want keyboard navigation to work for the column menu. I set up everything as it is shown in the documentation, I use kendo-grid-columnmenu-sort and a couple of kendo-grid-columnmenu-item components and pass the reference of each one to the kendoGridColumnMenuItem input. However, when I open the menu and try to use tab, the focus only goes between sort items and never reaches my custom items (kendo-grid-columnmenu-item).

### Steps To Reproduce

Stackblitz: https://angular-vcnfqtkp-k7n6hozi.stackblitz.io/

1. Open column menu on any header column.
2. Press tab a few times
3. Notice that focus goes between sort items only, never reaches "Hide Column" item

### Screenshots or video

"Hide Column" item never gets focus state

<img width="312" height="160" alt="Image" src="https://github.com/user-attachments/assets/1fa6830a-5d2c-4efc-b653-f36ad5d33069" />

### Actual Behavior

1. Open column menu on any header column.
2. Press tab.
3. Press tab again.
4. The focus goes back to "Sort Ascending" item.

### Expected Behavior

1. Open column menu on any header column.
2. Press tab.
3. Press tab again.
4. The focus goes to "Hide Column" item

### Browser

Chrome

### Browser version

latest

### OS type

MacOS
